synapse/storage/databases/main/relations.py

```diff
@@ -287,14 +296,71 @@ async def get_aggregation_groups_for_event(
         def _get_aggregation_groups_for_event_txn(
             txn: LoggingTransaction,
         ) -> List[JsonDict]:
-            txn.execute(sql, where_args)
+            txn.execute(sql, args)
 
             return [{"type": row[0], "key": row[1], "count": row[2]} for row in txn]
 
         return await self.db_pool.runInteraction(
             "get_aggregation_groups_for_event", _get_aggregation_groups_for_event_txn
         )
 
+    async def get_aggregation_groups_for_users(
```

Comment:
Not sure if this needs caching?

Reply:
I don't think these need caching, given that the common case will be a no-op anyway. We can always revisit it if we we see it taking time